### DISCUSSION TITLE
I am waiting for an update to support the MSX Adapter.

### DISCUSSION THREAD

**Discussion by Oswalds99** [sanni/cartreader#820] [General Chat]
I have made an MSX adapter and am waiting for update to support the MSX.
Forgive me for writing because I couldn't wait.
Thank you for this wonderful project!

**Comment by sanni** [comment]
You can update like described here: https://github.com/sanni/cartreader/wiki/Troubleshooting#updating-to-the-latest-github-code

**Reply by Oswalds99** [reply]
I will try with the latest sketch.
Thank you.

**Reply by PsyK0p4T** [reply]
I can confirm that the adapter + current dev branch code works.
4 MSX carts dumped fine so far.

**Reply by Oswalds99** [reply]
I was able to write the latest sketch.
and able to dump MSX.bin from cartridge.
I renamed it to xxx.rom and was able to confirm the operation with the emulator.
thank you so much!

**Reply by PsyK0p4T** [reply]
Don't hesitate to submit your dumps to No-Intro/Datomatic.
The MSX set is totally unverified at the moment, and we know that a lot of undumpeds/revisions are still around.
